package.json

```diff
@@ -92,6 +92,9 @@
 					"eslint-plugin/require-meta-docs-url": "off"
 				}
 			}
-		]
+		],
+		"rules": {
+			"import/order": "off"
+		}
```

Comment:
This is really weird,  see

https://github.com/voxpelli/eslint-plugin-unicorn/runs/437762645?check_suite_focus=true

Reply:
https://github.com/benmosher/eslint-plugin-import/pull/1651